EasyCommands.Tests/ScriptTests/FunctionalTests/SimpleComparisonsVectorTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+
+namespace EasyCommands.Tests.ScriptTests {
+    [TestClass]
+    public class SimpleComparisonsVectorTests {
+        [TestMethod]
+        public void EqualTo() {
+            var lines = new List<String> {
+                "x == 1:0:0",
```

Comment:
i think the tests will read a little better without using the built in "x,y,z" variables.  Can we make these explicit vectors instead?

Reply:
I am not sure how much the script likes comparisons with explicit left hand sides for vector, color and list.
Something like 
```
if [-1,2] contains var
    ...
```
throws an `key not found in dictionary` exception.

Reply:
huh, that sounds like a bug.  I'm curious to look into this one more